command-runner/runner.go

```diff
@@ -2,7 +2,7 @@ package command_runner
 
 import (
```

Comment:
Why do you need struct for it? seems to me as static utility function, which should be located under utils, there aren't any data members (except from logger).